packages/shared/src/hooks/post/useSmartTitle.ts

```diff
@@ -46,32 +51,36 @@ export const useSmartTitle = (post: Post): UseSmartTitle => {
 
   const { data: smartTitle, refetch } = useQuery({
     queryKey: key,
-    queryFn: async () => {
-      let title = post?.title || post?.sharedPost?.title;
+    queryFn: async (): Promise<PostSmartTitle> => {
+      const titleRecord = {
+        title: post?.title || post?.sharedPost?.title,
+        translation: post.translation,
+      };
```

Comment:
Refactored this so it also has `translation` property, this is because we have to manage translated stuff under title query key vs original post, because now they show separate states (smart title vs normal title depending on clickbait being active).